addons/sys_core/fnc_updateSelf.sqf

```diff
@@ -16,14 +16,16 @@
  */
 
 // ref bug: http://feedback.arma3.com/view.php?id=15580
-private _projectPos1 = ATLtoASL positionCameraToWorld [0,0,0];
+private _projectPos = ATLtoASL positionCameraToWorld [0, 0, 0];
 
-private _projectPos2 = ATLtoASL positionCameraToWorld [0,0,1];
-
-ACRE_LISTENER_DIR = _projectPos2 vectorDiff _projectPos1;
+ACRE_LISTENER_DIR = if (EGVAR(sys_zeus,zeusCommunicateViaCamera) && {call FUNC(inZeus)}) then {
```

Comment:
Same as above, but I could let it slide due to performance reasons in this case (both cases have commands).

Reply:
@TheMagnetar ?

Reply:
I tend to like inline ifs @jonpas ... but i agree on the readability part

Reply:
Reverted